android-build/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM ubuntu:22.04
+FROM ubuntu:23.04
```

Comment:
can you explain why we need not LTS version of ubuntu? is there any reason why 22.04 is not enough?

Reply:
Mea culpa. Fixed. Tested.